File: core/src/main/java/me/devsnox/planetsystem/core/database/PlanetDAO.java
package me.devsnox.planetsystem.core.database;

import org.mongodb.morphia.Datastore;
import org.mongodb.morphia.dao.BasicDAO;

public class PlanetDAO extends BasicDAO<DatabasePlanet, String> {

    public PlanetDAO(final Class<DatabasePlanet> entityClass, final Datastore ds) {
        super(entityClass, ds);
    }
}

File: core/src/main/kotlin/de/astride/planetsystem/core/database/PlanetDAO.kt
/*
 * © Copyright - Astride UG (haftungsbeschränkt) 2018 - 2019.
 */

package de.astride.planetsystem.core.database

import de.astride.planetsystem.core.database.entities.BasicOfflinePlanet
import xyz.morphia.Datastore
import xyz.morphia.dao.BasicDAO

class PlanetDAO(entityClass: Class<BasicOfflinePlanet>, ds: Datastore) :
    BasicDAO<BasicOfflinePlanet, String>(entityClass, ds)
